ACM_General/sigs/managers.py

```diff
@@ -4,20 +4,23 @@
 
 class SIGManager(models.Manager):
     """
-    TODO: Docstring
+    This class contains functions to test all functions of SIG's
     """
     use_in_migrations = True
 
     def get_by_natural_key(self, id):
         """
-        TODO: Docstring
+        Gets the SIG by it's id
+
+        :param id: The id of the SIG 
+        :type id: int
+        :rtype: None
+        :return: None
         """
         return self.get(id=id)
 
     def _create_sig(self, **kwargs):
-        """
-        TODO: Docstring
-        """
+        
```

Comment:
Add docstring

Reply:
I did, but the docstring does not render in Sphinx.

Reply:
It wont until Zach's `fix/sphinx-docs` branch is merged